identity/client/src/pages/groups/List.tsx

```diff
@@ -8,31 +8,79 @@
 import { FC, useState } from "react";
 import useTranslate from "src/utility/localization";
 import { useApi } from "src/utility/api/hooks";
-import Page from "src/components/layout/Page";
+import Page, { PageTitle } from "src/components/layout/Page";
 import EntityList, {
   DocumentationDescription,
 } from "src/components/entityList";
-import { DocumentationLink } from "src/components/documentation";
+import {
+  documentationHref,
+  DocumentationLink,
+} from "src/components/documentation";
 import { getGroups, Group } from "src/utility/api/groups";
 import { useNavigate } from "react-router";
 import { TranslatedErrorInlineNotification } from "src/components/notifications/InlineNotification";
+import { useEntityModal } from "src/components/modal";
+import useModal from "src/components/modal/useModal";
```

Comment:
Just a comment, no changes needed: these two imports can be combined

```
import useModal, { useEntityModal } from "src/components/modal/useModal";
```

I see that having the hooks exported from different places breaks the IDE import optimization. There is no need to change the imports here, we can do some light refactoring to have clear imports here later.